src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Books from './pages/Books';
+import Categories from './pages/Categories';
+import './styles/App.css';
+
+function App() {
+  return (
+    <main className="App">
+      <Navbar />
+      <Routes>
+        <Route path="/books">
+          <Books />
+        </Route>
+        <Route path="/categories">
+          <Categories />
+        </Route>
+        <Route path="/">
+          <Navigate to="/books" />
+        </Route>
+      </Routes>
```

Comment:
- [ ] When your app is run it logs some errors relating to your routing that prevents your app from displaying anything at all (see image). Kindly fix that.

  **SUGGESTION:** You can configure your routes like in the snippet below and remember to configure your **Navbar.js** links accordingly.
  ```
  <Routes>
    <Route path="/" element={<Books />} />
    <Route path="/categories" element={<Categories />} />
  </Routes>
  ```


  ![varay-route](https://user-images.githubusercontent.com/36542986/154920801-916099d4-3f4b-43f9-b2bb-84ad6d96dc75.jpg)

Reply:
Thank you Sir for the suggestion.